### ISSUE TITLE
Differing FID Score

### ISSUE THREAD

**Issue by AnqiGou** [tyxsspa/AnyText#141]
I referred to the documentation for the measurement of the FID score, first I used the ckpt1.1 model to generate 1000 test images, and calculated the FID score with the laion-40k images in the FID folder, and the score was 57.86, which is very different from the 33.54 in the paper. The results of OCR and NED are not much different. What could be the reason for this?

**Comment by tyxsspa** [comment]
Hi, 
Are you using `eval/eval_fid.sh` to calculate FID?
The generate 1000 images all in English right? (if they are in Chinese, dont compare FID with laion-40k)

**Comment by AnqiGou** [comment]
Yes, all the 1000  images are in English (with test1k.json file), and I calculate FID with eval/eval_fid.sh. I have on idea why the fid score is so much worse than yours, are there any details that need to be paid attention to?
